testbed/settings/development.py

```diff
@@ -14,9 +14,23 @@
 ALLOWED_HOSTS = env.list("DJANGO_ALLOWED_HOSTS", default=["localhost", "127.0.0.1"])
 
 # Seeding settings
-SEED_ADMIN_USERNAME = ("admin",)
-SEED_ADMIN_EMAIL = ("admin@testing.com",)
-SEED_ADMIN_PASSWORD = ("admin123",)
+SEED_ADMIN_USERNAME = "admin"
+SEED_ADMIN_EMAIL = "admin@seeding.com"
```

Comment:
FYI - A url with "example.com" or "example.org" will be guaranteed on the Internet never to actually send the mail - so we could use "admin@seeding.example.com"